src/main/java/com/myname/mymodid/CommonProxy.java

```diff
@@ -20,7 +23,14 @@ public void init(FMLInitializationEvent event) {
 
     // postInit "Handle interaction with other mods, complete your setup based on this."
     public void postInit(FMLPostInitializationEvent event) {
+        // Access originally package-private subclass
+        GuiScreenBook.NextPageButton button = null;
 
+        // Access originally private field
+        Material air = Blocks.air.blockMaterial;
+
+        // Access originally protected method
+        Blocks.air.disableStats();
```

Comment:
Do we have them enabled by default? Alternatively, we could comment them out and deactivate the option in `gradle.properties`

Reply:
I would rather go with creating a separate page explaining how reflections are activated and used in 1.7.10 with code snippet(s)